test/utils/probability/test_lin_ess.py

```diff
@@ -462,3 +462,45 @@ def test_multivariate(self):
             self.assertEqual(X_high_d.shape, torch.Size([16, d]))
             self.assertTrue(sampler._is_feasible(X_high_d.T).all())
             self.assertEqual(sampler.lifetime_samples, num_samples)
+
+    def test_batch_mcmc(self):
+        # all random seeds in [0, 99] should pass
+        torch.manual_seed(torch.randint(100, (1,)))
+
+        d = 5
+
+        for dtype, atol in zip((torch.float, torch.double), (2e-5, 1e-12)):
```

Comment:
`atol` isn't used here. Is this suggestion right? If so, this looks good to go :)

```suggestion
        for dtype in (torch.float, torch.double):
```

Reply:
By the way, I merged in main and made a minor change to a comment, so you'll need to either pull down those changes or make further changes directly from the UI.

Reply:
> `atol` isn't used here. Is this suggestion right? If so, this looks good to go :)

Yep, `atol` is not used. It should be removed.